h2o-docs/src/product/data-science/glm.rst

```diff
@@ -1341,6 +1341,223 @@ You can extract the columns in the Coefficients Table by specifying ``names``, `
+
+.. note::
+   
+   You must estimate the dispersion parameter. During initialization, ``compute_p_values`` and ``remove_collinear_columns`` must be set to ``True`` to facilitate this estimation process. Also, ``dispersion_parameter_method`` must be set to ``"pearson"``.
+
+**Binomial**:
+
+.. math::
+   
+   l \big(\mu_i (\beta); y_i, w_i \big) = w_i \big(y_i \log \{ \mu_i \} + (1-y_i) \log \{ 1-\mu_i \} \big)
+
+where
+
+- :math:`\mu_i` is the probability of 1
+- :math:`y_i` is the real value of the target variable
+
+**Quasibinomial**:
+
+- If the predicted value equals :math:`y_i`, log likelihood is 0
+- If :math:`\mu_i >1` then :math:`l(\mu_i (\beta); y_i) = y_i \log \{ \mu_i \}`
+- Otherwise, :math:`l(\mu_i (\beta); y_i) = y_i \log \{ \mu_i \} + (1-y_i) \log \{ 1- \mu_i \}` where
+   
+   - :math:`mu_i` is the probability of 1
+   - :math:`y_i` is the real value of the target variable
+
+**Fractional Binomial**:
+
+.. math::
+   
+   l(\mu_i (\beta); y_i) = w_i \Big(y_i \times \log \big(\frac{y_i}{\mu_i} \big) + (1-y_i) \times \log \big(\frac{1-y_i}{1-\mu_i} \big) \Big)
+
+where
+
+- :math:`\mu_i` is the probability of 1
+- :math:`y_i` is the real value of the target variable
+
+**Poisson**:
+
+.. math::
+   
+   l(\mu_i (\beta); y_i) = w_i \big(y_i \times \log (\mu_i) - \mu_i - \log (\Gamma (y_i +1)) \big)
+
+where
+
+- :math:`\mu_i` is the prediction
+- :math:`y_i` is the real value of the target variable
+
+**Negative Binomial**:
+
+.. math::
+   
+   l(\mu_i (\beta); y_i, w_i) = y_i \log \big(\frac{k \mu}{w_i} \big) - \big(y_i + \frac{w_i}{k} \big) \log \big(1 + \frac{k \mu}{w_i} \big) + \log \Big(\frac{\Gamma \big( y_i = \frac{w_i}{k} \big)} {\Gamma (y_i +1) \Gamma \big(\frac{w_i}{k}\big)} \Big)
+
+where
+
+- :math:`\mu_i` is the prediction
+- :math:`y_i` is the real value of the target variable
+- :math:`k = \frac{1}{\phi}` is the dispersion parameter estimation
+
+.. note::
+   
+   You must estimate the dispersion parameter. During initialization, ``compute_p_values`` and ``remove_collinear_columns`` must be set to ``True`` to facilitate this estimation process. Also, ``dispersion_parameter_method`` must be set to ``"ml"``.
+
+   Since ``dispersion_parameter_method="ml"`` only works without regularization, you disable regularization by setting ``lambda`` equal to an array of ``0.0``.
+
+**Gamma**:
+
+.. math::
+   
+   l(\mu_i (\beta); y_i, w_i) = \frac{w_i}{\phi} \log \big( \frac{w_i y_i}{\phi \mu_i} \big) - \frac{w_i y_i}{\phi \mu_i} - \log (y_i) - \log \big(\Gamma \big(\frac{w)i}{\phi} \big) \big)
+
+where
+
+- :math:`\mu_i` is the prediction
+- :math:`y_i` is the real value of the target variable
+- :math:`\phi` is the dispersion parameter estimation
+
+.. note::
+   
+   You must estimate the dispersion parameter. During initialization, ``compute_p_values`` and ``remove_collinear_columns`` must be set to ``True`` to facilitate this estimation process. Also, ``dispersion_parameter_method`` must be set to ``"ml"``.
```

Comment:
Please change the note:

\emph{Note:} For Gamma family, the dispersion parameter estimate is needed. When parameter calc_like is set to true, the parameters compute_p_values and remove_collinear_columns are set to true for you. By default, the parameter dispersion_parameter_method is set to \texttt{"pearson"}. However, dispersion_parameter_method can be set to \texttt{"deviance"} or \texttt{"ml"} by the user.